client/landing/gutenboarding/hooks/use-site-title.ts

```diff
@@ -13,11 +13,14 @@ import usePodcastTitle from './use-podcast-title';
 export default function useSiteTitle(): void {
 	const { setSiteTitle } = useDispatch( ONBOARD_STORE );
 	const podcastTitle = usePodcastTitle();
+	const { siteTitle } = useSelect( ( select ) => ( {
```

Comment:
This hook call is structured this way to return and check the value of the actual title, vs. the function.